### PR TITLE
add the remote signer

### PR DESCRIPTION
**Pull request by mastereng12**
## Context

Closes: #118

---

## Reviewers

- @Lagrange-Labs/lsc-core-dev

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -344,8 +336,8 @@ func deregisterOperator(c *cli.Context) error {
 	}
 
 	// deregister the operator from the committee
-	logger.Infof("Deregistering with sign address: %s", cliCfg.SignerAddress)
-	chainOps, err := utils.NewChainOps(network, cliCfg.EthereumRPCURL, cliCfg.OperatorPrivKey)
+	logger.Infof("Deregistering with sign key id: %s", cliCfg.SignerKeyAccountID)
```
Comment (kashishshah) [inline cmd/main.go:339]: Nitpick
```suggestion
	logger.Infof("Deregistering with signer key id: %s", cliCfg.SignerKeyAccountID)
```

**Comment by kashishshah** [comment]
What about changes to `config.toml` file?

**Comment by mastereng12** [comment]
@mastereng12 , created the test epic, need to keep this PR unmerged until the test.

File: cmd/main.go (hunk 1 above)
Comment (mastereng12) [reply cmd/main.go:339]: I'd like to keep `sign key` since we had the `signer daemon` term

**Comment by mastereng12** [comment]
ok, I finished the local e2e testing

**Pull request merged**